src/styles/theme.ts

```diff
@@ -109,6 +109,23 @@ const initTheme = (darkMode: boolean) => {
       },
     },
     components: {
+      MuiCssBaseline: {
+        styleOverrides: `
+          @font-face {
```

Comment:
Why does it have to be in the MUI theme and not in the global CSS? What's the difference?

Reply:
I don't think there is a difference. They state
> In order to globally define Raleway as a font face, the [CssBaseline](https://mui.com/material-ui/react-css-baseline/) component can be used (or any other CSS solution of your choice).

Reply:
Should I revert it? Also find it more readable in `globals.css`

Reply:
If the font is defined in the theme, it can be used by others, e.g. when we export it in SRC.

Reply:
Oh yes. Didn't think of that.

Reply:
Yes, no unnecessary changes is our motto.